includes/functions.php

```diff
@@ -2269,7 +2294,7 @@ function tsml_date_localised($format, $timestamp = null)
 	// From https://mediarealm.com.au/
 
 	$tz_string = get_option('timezone_string');
-	$tz_offset = get_option('gmt_offset', 0);
+	$tz_offset = get_option('gmt_offset', -6);
```

Comment:
my eye was drawn to the `-6` here but im not sure i undertand the purpose of this `tsml_strtotime` function as a whole, are we sure we need it?

Reply:
Good catch. I was trying something and it didn't pan out. I need to reset to 0 or better yet, if it's not being used anywhere (which I think is the case) then just get rid of it.